--- a/barbican/cmd/keystone_listener.py	2016-01-18 16:05:30.934396751 +0000
+++ b/barbican/cmd/keystone_listener.py	2016-04-29 17:30:44.855698125 +0000
@@ -52,17 +52,20 @@
     sys.exit(returncode)
 
 
-if __name__ == '__main__':
+#if __name__ == '__main__':
+def main():
     try:
         config.setup_remote_pydev_debug()
+
+        CONF = config.CONF
+
         # Import and configure logging.
-        log.setup('barbican')
+        log.setup(CONF, 'barbican')
 
         LOG = log.getLogger(__name__)
         LOG.info("Booting up Barbican Keystone listener node...")
 
         # Queuing initialization
-        CONF = config.CONF
         queue.init(CONF)
 
         if getattr(getattr(CONF, queue.KS_NOTIFICATIONS_GRP_NAME), 'enable'):
